File: Junit/Poligonos/src/Poligonos/Ponto2D.java
package Poligonos;

public class Ponto2D {
	private double x,y;

	public Ponto2D(){
		x=0;
		y=0; 
	}

	public Ponto2D(double x, double y){
		this.x=x;
		this.y=y; 
	}

	public double getX(){
		return x; 
	}

	public double getY(){
		return y; 
	}

	public void setX(double x){
		this.x=x; 
	}
	
	public void setY(double y) {
		this.y=y;
	}
	
	public static double DistPontos(Ponto2D p1, Ponto2D p2){
		return (Math.sqrt(Math.pow((p2.x-p1.x),2) + Math.pow((p2.y-p1.y), 2)));
	}
}

File: Poligonos/Ponto2D.py
import math
class Ponto2D(object):
    def __init__(self, x=0, y=0):
        self.x = x
        self.y = y

    def setX(self, x):
        self.x = x

    def setY(self, y):
        self.y = y

    def getX(self, x):
        return self.x

    def getY(self, y):
        return self.y

    def calcDistance(p1,p2):
        distX = abs(p1.x - p2.x)
        distY = abs(p1.y - p2.y)
        return math.sqrt(distX^2 + distY^2);